docs/source/quick_start.rst

```diff
@@ -3,23 +3,432 @@ Quick start
+            ],
+            "redcap.UPENN": [
+              "UPENN"
+            ]
+          },
+          ...,
+          "PronetSL": {
+            "redcap.Pronet": [
+              "Pronet"
+            ],
+            "redcap.UPENN": [
+              "UPENN"
+            ]
+          }
+        },
+        "SECRETS": {
+          "PronetLA": "LOCHNESS_SECRETS",
+          ...,
+        }
+      },
+      "redcap.UPENN": {
+        "URL": "*****",
+        "API_TOKEN": {
+          "UPENN": "*****"
+        }
+      },
+      "redcap.Pronet": {
+        "URL": "*****",
+        "API_TOKEN": {
+          "Pronet": "*****"
+        }
+      },
+      "xnat.PronetLA": {
+        "URL": "*****",
+        "USERNAME": "*****",
+        "PASSWORD": "*****"
+      },
+      ...,
+      "box.PronetLA": {
+        "CLIENT_ID": "*****",
+        "CLIENT_SECRET": "*****",
+        "ENTERPRISE_ID": "*****"
+      },
+      ...,
+      "mindlamp.PronetLA": {
+        "URL": "*****",
+        "ACCESS_KEY": "*****",
+        "SECRET_KEY": "*****"
+      },
+      ...,
+    }
+
+    
+Example of completed ``lochness.json`` ::
+
+    {
+      "lochness": {
+        "REDCAP": {
+          "PronetLA": {
+            "redcap.Pronet": [
+              "Pronet"
+            ],
+            "redcap.UPENN": [
+              "UPENN"
+            ]
+          },
+          ...,
+          "PronetSL": {
+            "redcap.Pronet": [
+              "Pronet"
+            ],
+            "redcap.UPENN": [
+              "UPENN"
+            ]
+          }
+        },
+        "SECRETS": {
+          "PronetLA": "LOCHNESS_SECRETS",
+          ...,
+        }
+      },
+      "redcap.UPENN": {
+        "URL": "https://redcap.med.upenn.edu",
+        "API_TOKEN": {
+          "UPENN": "BC6BEF2D2369BC8FE1233CAAAB20378D"
+        }
+      },
+      "redcap.Pronet": {
+        "URL": "https://redcapynh-p11.ynhh.org"
+        "API_TOKEN": {
+          "Pronet": "AFBDCCD55934EE947A388541EED6A216"
+        }
+      },
+      "xnat.PronetLA": {
+        "URL": "https://xnat.med.yale.edu",
+        "USERNAME": "kcho",
+        "PASSWORD": "whrkddlr8*90"
+      },
+      ...,
+      "box.PronetLA": {
+        "CLIENT_ID": "e19fltqp9f9ftv4dydqjius4w20072cr",
+        "CLIENT_SECRET": "LrkDwYZvA49Q4dXVGv3g4aaSy4SQRobz",
+        "ENTERPRISE_ID": "756591"
+      },
+      ...,
+      "mindlamp.PronetLA": {
+        "URL": "mindlamp.orygen.org.au",
+        "ACCESS_KEY": "kcho",
+        "SECRET_KEY": "0c5b0a5af972b2a1b2d6cd299dc37703c22e8ddd5dfd15f0d83ca7a1cb8bcce7"
+      },
+      ...,
+    }
+
+
+
+3. Encrypt ``lochness.json`` by running ::
+
+    $ bash 2_sync_command.sh
+
+Then remove ``lochness.json`` for security ::
+
+    $ rm lochness.json
+
+
+
+4. Edit ``config.yml``::
+
+    $ vim config.yml
+
+
+Edit AWS s3 bucket name and root directory ::
+
+    AWS_BUCKET_NAME: pronet-test
+    AWS_BUCKET_ROOT: TEST_PHOENIX_ROOT_PRONET
+
+
+Edit ``base`` field for Box structure ::
```

Comment:
We are saying edit `config.yml` by editing AWS and Box sections. So can we have two bullets before lines 246 and 252?